app/screens/HomePage/Filter.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  View, StyleSheet, ScrollView,
+} from 'react-native';
+import { ButtonGroup, Button, } from 'react-native-elements';
+import { ArrowSmallUpIcon as ArrowSmallUpIconOutline, ArrowSmallDownIcon as ArrowSmallDownIconOutline } from "react-native-heroicons/outline";
+import COLORS from "../../constants/colors";
```

Comment:
Colors is exported as an object.
```suggestion
import { COLORS } from "../../constants/colors";